### PR TITLE
Add conflict behavior query parameter notes

### PR DESCRIPTION
**Pull request by micesari**

### CODE REVIEW

**Review by chackman — APPROVED** [review]

File: api-reference/v1.0/api/driveitem-copy.md [hunk 1]
```diff
@@ -47,6 +47,18 @@ In the request body, provide a JSON object with the following parameters.
...
 **Note:** The _parentReference_ should include the `driveId` and `id` parameters for the target folder.
 
+## Optional query parameters
+
+This method supports the `@microsoft.graph.conflictBehavior` query parameter to customize the behavior when a conflict occurs.
+
+| Value           | Description                                    |
+|:----------------|:---------------------------------------------- |
+| replace         | Overwrite existing item at the target site.    |
+| rename          | Rename the item.                               |
```
Comment (learafa) [inline api-reference/v1.0/api/driveitem-copy.md:57]: There is also fail as well. Here is the link in graph that has an example of the conflictBehaviors: https://docs.microsoft.com/en-us/graph/api/resources/driveitem?view=graph-rest-1.0.

Comment (micesari) [reply api-reference/v1.0/api/driveitem-copy.md:58]: Added

File: api-reference/v1.0/api/driveitem-copy.md [hunk 2]
```diff
@@ -35,7 +34,19 @@ POST /sites/{siteId}/drive/items/{itemId}/copy
 POST /users/{userId}/drive/items/{itemId}/copy
 ```
 
-### Request body
+## Query parameters
```
Comment (Lauragra) [inline api-reference/v1.0/api/driveitem-copy.md:37]: Please make this update to the file in the /beta folder as well.

Comment (k-tsoi) [reply api-reference/v1.0/api/driveitem-copy.md]: updated to reflect beta changes
https://github.com/microsoftgraph/microsoft-graph-docs/pull/14433/files#diff-618537577de0a5d7395d4e5bbc7e18f257a3ec8718eb83a7adc2e776995250f7

**Review by isvargasmsft — APPROVED** [review]

**Review by Lauragra — APPROVED** [review]

**Pull request merged**